starknet-testnet/server.py

```diff
@@ -63,6 +70,8 @@ async def invoke():
         print("-----Invoke info-----")
         print(execution_info)
         print("----------\n")
+        steps_in_function_invoke(data["function"], execution_info)
```

Comment:
Would it be possible (and not too time consuming) to add a flag to the testnet such that the benchmarking is only done when the flag is passed, and have the CI pass the flag? That way when we're running lots of behaviour tests during development it won't get slowed down